arm-customer-insights/2016-01-01/examples/ProfilesCreateOrUpdate.json

```diff
@@ -0,0 +1,376 @@
+{
+	"parameters": {
+		"hubName": "sdkTestHub",
+		"resourceGroupName": "TestHubRG",
+		"profileName": "TestProfileType396",
+		"api-version": "2016-01-01",
+		"subscriptionId": "subid",
+		"parameters": {
+			"location": "West US",
+			"properties": {
+				"strongIds": [{
+					"keyPropertyNames": ["Id",
+					"SavingAccountBalance"],
+					"strongIdName": "Id"
+				},
+				{
+					"keyPropertyNames": ["ProfileId",
+					"LastName"],
+					"strongIdName": "ProfileId"
+				}],
+				"apiEntitySetName": "TestProfileType396",
+				"entityType": "Profile",
+				"fields": [{
+					"fieldName": "Id",
+					"fieldType": "Edm.String",
+					"isArray": false,
+					"isRequired": true
+				},
+				{
+					"fieldName": "ProfileId",
+					"fieldType": "Edm.String",
+					"isArray": false,
+					"isRequired": true
+				},
+				{
+					"fieldName": "LastName",
+					"fieldType": "Edm.String",
+					"isArray": false,
+					"isRequired": true
+				},
+				{
+					"fieldName": "TestProfileType396",
+					"fieldType": "Edm.String",
+					"isArray": false,
+					"isRequired": true
+				},
+				{
+					"fieldName": "SavingAccountBalance",
+					"fieldType": "Edm.Int32",
+					"isArray": false,
+					"isRequired": true
+				}],
+				"schemaItemTypeLink": "SchemaItemTypeLink",
+				"smallImage": "\\\\Images\\\\smallImage",
+				"mediumImage": "\\\\Images\\\\MediumImage",
+				"largeImage": "\\\\Images\\\\LargeImage"
+			}
+		}
+	},
+	"responses": {
+		"201": {
+			"body": {
+				{
```

Comment:
unnecessary curly braces. The "body" key points to an object, no need to wrap it again inside curly braces. please remove the opening and closing curly braces.